src/test/unit/algorithms/AnomalyTest.cpp

```diff
@@ -131,7 +132,25 @@ TEST(Anomaly, SelectModePure) {
   ASSERT_FLOAT_EQ(a.compute(active, predicted), 2.0f / 3.0f);
 };
 
+TEST(Anomaly, SerializationPure)
+{
+  Anomaly a{0, AnomalyMode::PURE, 0};
+  Anomaly b;
+
+ std::vector<UInt> active = {2, 3, 6};
+  std::vector<UInt> predicted = {3, 5, 7};
+  a.compute(active, predicted);
+
+
+  std::stringstream ss;
+  a.saveToStream_ar(ss);
+  b.loadFromStream_ar(ss);
+  //EXPECT_EQ(a, b);
```

Comment:
TODO actually test it

Reply:
> TODO actually test it

Finished debugging the operator== for this set.  It now works (at least it does locally).